Apsitvarkom.UnitTests/DataAccess/PollutedLocationsFileRepositoryTests.cs

```diff
@@ -1,13 +1,27 @@
-﻿using Apsitvarkom.DataAccess;
+﻿using System.Text.Json;
+using Apsitvarkom.DataAccess;
 using static Apsitvarkom.Models.Enumerations;
 
 namespace Apsitvarkom.UnitTests.DataAccess;
 
-public class PollutedLocationsFileStoreTests
+public class PollutedLocationsFileRepositoryTests
 {
+    // Existing mock data file, containing invalid json data.
+    private string m_invalidDataSourcePath;
+
+    // Existing mock data file, containing three valid instances with unique property values.
+    private string m_validDataSourcePath;
+
+    [OneTimeSetUp]
+    public void OneTimeSetUp()
+    {
+        m_validDataSourcePath = Path.Combine("DataAccess", "PollutedLocationDTOMockValid.json");
+        m_invalidDataSourcePath = Path.Combine("DataAccess", "PollutedLocationDTOMockInvalid.json");
+    }
```

Comment:
I believe you don't even have to have this `OneTimeSetUp` here. 

`private static readonly string InvalidDataSourcePath = Path.Combine("DataAccess", "PollutedLocationDTOMockValid.json")`

I think should work